test/unit/lib/fetch-system-codes.js

```diff
@@ -46,8 +46,9 @@ describe('lib/middleware/fetch-system-codes', () => {
 					count: 'Pages: 3, Items: 4'
 				},
 				body: [
-					{value: 'foo'},
-					{value: 'bar'}
+					{systemCode: 'foo'},
+					{noSystemCode: 'this should not appear'},
+					{systemCode: 'bar'}
 				]
```

Comment:
Due to above issue with the "systemCode" field I recommend changing these mockups to populate "dataItemID" since that is the real system code field.